quickwit/rest-api-tests/run_tests.py

```diff
@@ -85,7 +85,7 @@ def run_request_step(method, step, previous_result):
         step["headers"] = {'user-agent': 'my-app/0.0.1'}
     method_req = getattr(requests, method.lower())
     endpoint = step.get("endpoint", "")
-    url = step["api_root"] + endpoint
+    url = os.path.join(step["api_root"], endpoint)
```

Comment:
hm... this is a url here. not a filepath. For instance, I think os.path.join uses "\" as a delimiter on windows.